support/scheduler-client/client.go

```diff
@@ -0,0 +1,89 @@
+//
+// Copyright (c) 2017 Tencent
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package scheduler
+
+import (
+	"bytes"
+	"github.com/edgexfoundry/edgex-go/core/domain/models"
+	"github.com/edgexfoundry/edgex-go/support/logging-client"
+	"io"
+	"net/http"
+)
+
+const (
+	SchedulerClientName = "support-scheduler-client"
+)
+
+// Common http const
+const (
+	HttpPostMethod     = "POST"
+	ContentType        = "Content-Type"
+	ContentTypeJsonVal = "application/json"
+)
+
+var loggingClient logger.LoggingClient = logger.NewClient(SchedulerClientName, false, "")
+
+// Struct to represent the scheduler client
+type SchedulerClient struct {
+	RemoteScheduleUrl      string
+	RemoteScheduleEventUrl string
```

Comment:
The only needed configuration should be the hostname and port, not urls. The client should know how to create the complete url from the hostname and port.

Reply:
@feclare I agree this opinion, I just followed the notification's client, I will change this.